pandora-client-web/src/ui/components/chat/commands.ts

```diff
@@ -307,4 +308,62 @@ export const COMMANDS: readonly IClientCommand<ICommandExecutionContextClient>[]
 				return true;
 			}),
 	},
+	// Commands to change the player's pose
```

Comment:
there is also a turn command it may worth moving it down here

also you could use:
```ts
//#region Commands to change the player's pose
//#endregion
```

that way VSCode allow a nice collapse option

Reply:
Good idea. I'll use it when I add the more sophisticated pose commands.